plugins/rest/rest_test.go

```diff
@@ -41,8 +41,31 @@ import (
 
 const keyID = "key1"
 
-func TestNew(t *testing.T) {
+func TestAuthPluginWithNoAuthPluginLookup(t *testing.T) {
+	authPlugin := "anything"
+	cfg := Config{
+		Credentials: struct {
```

Comment:
Do we need to list all the fields again here...? 🤔

Reply:
I tried to only pass the field plugin but it fails because types aren't the same.